akka-docs/src/main/paradox/general/actors.md

```diff
@@ -158,12 +158,9 @@ their parent.
 
 ## Supervisor Strategy
 
-The final piece of an actor is its strategy for handling faults of its
-children. Fault handling is then done transparently by Akka, applying one
-of the strategies described in @ref:[Fault Tolerance](../typed/fault-tolerance.md)
-for each incoming failure.
-As this strategy is fundamental to how an actor system is structured, it
-cannot be changed once an actor has been created.
+The final piece of an actor is its a strategy for handling unexpected exceptions - failures. 
+Fault handling is then done transparently by Akka, applying one of the strategies described 
+in @ref:[Fault Tolerance](../typed/fault-tolerance.md) for each failure.
```

Comment:
I wonder if should mention that the supervision strategy is typically defined by the parent actor when it's spawning a child?

Even if the supervision logic isn't executed by the parent actor any more I think it's often a good recommendation to put the supervision strategy as a responsibility of the parent when spawning (rather than in the Behavior factory method).

Reply:
I agree, I'll add that.

Reply:
Added